book/src/development/infrastructure/changelog_update.md

```diff
@@ -48,16 +48,13 @@ git log --oneline -- src/tools/clippy/ | grep -o "Merge commit '[a-f0-9]*' into
 Once you've got the correct commit range, run
 
 ```
-util/fetch_prs_between.sh commit1 commit2 > changes.txt
+src/tools/clippy/util/fetch_prs_between.sh start_commit end_commit > changes.txt
```

Comment:
This should be run in the Clippy repo.

```suggestion
util/fetch_prs_between.sh start_commit end_commit > changes.txt
```

Thanks for noting which is the start and end commit!